### PR TITLE
B/update n rewards rendering

### PR DESCRIPTION
**Pull request by specialist0x**

### CODE REVIEW

File: src/state/stake/stake-constants.ts [hunk 1]
```diff
@@ -112,10 +111,10 @@ const NULL_POOL: StakingTri = {
   apr: 0,
   nonTriAPRs: [],
   chefVersion: ChefVersions.V1,
-  doubleRewards: false,
+  hasNonTriRewards: false,
   inStaging: false,
   noTriRewards: false,
-  doubleRewardToken: dummyToken,
+  earnedNonTriRewards: [],
```
Comment (specialist0x) [inline src/state/stake/stake-constants.ts:117]: This makes adding pools easier, but if a user has farmed tokens that
1. haven't been claimed
2. rewards are turned off

They'll only know if they look at the farm card

**Review by rej156 — APPROVED** [review]
phat pr, lgtm
assuming it's all defensible code

**Comment by rej156** [comment]
request, could you use yarn commit after git add . next time for cleaner commit messages :P

**Comment by specialist0x** [comment]
This is a large PR (sorry):

- Using the new APR structure (https://cdn.trisolaris.io/datav2.json)
  - Removed need for
    - adding individual reward tokens
    - indicating if TRI rewards are enabled
  - Rewards keys like `dualRewardToken` and `noTriReward` are dynamically calculated from the AJAX response

- Updated methods to fetch token logos
- Update UI to support `n` reward tokens
- Updated `stake-constants` to remove superfluous information

**Comment by specialist0x** [comment]
I'll be squashing this PR when it's merged in

**Comment by specialist0x** [comment]
Left is dev, right is prod:
image
image

**Review by rej156 — APPROVED** [review]

**Review by memacius32 — APPROVED** [review]

**Comment by memacius32** [comment]
will rebase and merge after testing locally with wnear-bbt farm

**Pull request merged**